WooCommerce/StoreWidgets/StoreInfoProvider.swift

```diff
@@ -129,14 +133,41 @@ private extension StoreInfoProvider {
               let storeID = UserDefaults.group?[.defaultStoreID] as? Int64,
               let storeName = UserDefaults.group?[.defaultStoreName] as? String,
               let storeTimeZoneGMTOffset = UserDefaults.group?[.defaultStoreTimeZoneGMTOffset] as? Int,
-              let storeTimeZone = TimeZone(secondsFromGMT: storeTimeZoneGMTOffset) else {
+              let storeTimeZone = TimeZone(secondsFromGMT: storeTimeZoneGMTOffset),
+              let storeCurrencySettingsData = UserDefaults.group?[.defaultStoreCurrencySettings] as? Data,
+              let storeCurrencySettings = try? JSONDecoder().decode(CurrencySettings.self, from: storeCurrencySettingsData) else {
             return nil
         }
-        return Dependencies(authToken: authToken, storeID: storeID, storeName: storeName, storeTimeZone: storeTimeZone)
+        return Dependencies(authToken: authToken,
+                            storeID: storeID,
+                            storeName: storeName,
+                            storeTimeZone: storeTimeZone,
+                            storeCurrencySettings: storeCurrencySettings)
     }
 }
 
 private extension StoreInfoProvider {
+
+    func formattedAmountString(for amountValue: Decimal, with currencySettings: CurrencySettings?) -> String {
```

Comment:
I think these could be static, right? Since they don't access `self` at all.

Reply:
Yes, updated in 37cca09